doc/WELOTEC.md

```diff
@@ -0,0 +1,18 @@
+# Arrakis-pico-MK3
+
+- **Product Page:** https://www.welotec.com/de/industrie-pcs/
+- **BSP (dynamic layer):** [meta-welotec-bsp](https://github.com/omnect/meta-welotec-bsp)
```

Comment:
remove since not public?

Reply:
replace via https://github.com/yoctoproject/poky/blob/master/meta-yocto-bsp/conf/machine/genericx86-64.conf and resp. meta-yocto-bsp? ... we have to investigate

Reply:
removed and added a ToDo